server/webserver.go

```diff
@@ -85,58 +86,66 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 	//Set the broadcast config for creating onchain jobs.
 	mux.HandleFunc("/setBroadcastConfig", func(w http.ResponseWriter, r *http.Request) {
 		if err := r.ParseForm(); err != nil {
-			glog.Errorf("Parse Form Error: %v", err)
+			err = errors.Wrapf(err, "Parse form error")
+			glog.Error(err)
+			respondWith500(w, err.Error())
```

Comment:
`respondWith400` would be more appropriate?  Problem is with user's data, so it is a `BadRequest`, not `InternalServerError`

Reply:
fixed in 808250b for all user input related errors.